compiler/cpp/src/thrift/parse/t_field.h

```diff
@@ -68,7 +68,7 @@ class t_field : public t_doc {
 
   int32_t get_key() const { return key_; }
 
-  enum e_req { T_REQUIRED, T_OPTIONAL, T_OPT_IN_REQ_OUT };
+  enum e_req : unsigned { T_REQUIRED, T_OPTIONAL, T_OPT_IN_REQ_OUT };
```

Comment:
We still support C++03 and earlier compilation, so this won't work.  I know it's a burden, but I'm also on some projects that haven't moved to C++11 yet, amazing as it is.